### ISSUE TITLE
flyd.on subscriptions cannot be removed

### ISSUE THREAD

**Issue by teod** [paldepind/flyd#61]
Hello, 
   It's not possible to unsubscribe the .on listener on the flyd object. Shouldn't there be an .off method ? Is there a current workaround for this issue ?

**Comment by paldepind** [comment]
That is a good point. You're welcome to do a PR. Otherwise I'll look at it in 2-4 days.

**Comment by teod** [comment]
Is using the map method a good workaround ?
```javascript
    startStream() {
        "use strict";
        this.stream = flyd.map((smth)=> {
            ...
        }, SmthStream);
    }
    endStream() {
        "use strict";
        this.stream.end(true);
    }
```

**Comment by paldepind** [comment]
Yes. You can very well do that!

**Comment by paldepind** [comment]
Fixed by @gilbox in #64. I've updated the documentation.
